src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss

```diff
@@ -2,65 +2,86 @@
 @import "../../../../assets/stylesheets/mixins";
 @import "../../../../assets/stylesheets/color-variables";
 
-.comment-content{
-    width: $width100;
-    float: left;
-    padding: em(20) 0;
-    > div{
-        padding: 0;
+.comment-content {
+  width: $width100;
+  float: left;
+  padding: em(20) 0;
+  > div{
+      padding: 0;
+  }
+  .wi-comment-panel {
+    border-left: none;
+    border-right: none;
+    .panel-heading{
+      border: none;
     }
-    #wi-comment-add-comment {
-        /* Overrides pf's form-control style's height property */
-        /* The min-height is needed to avoid zero-height element */
-        /* The min-height is same as form-control's default height */
+    .wi-comment-input{
+      padding: em(20);
+      border-top: em(1) solid $color-pf-black-300;
+      #wi-comment-add-comment {
+      /* Overrides pf's form-control style's height property */
+      /* The min-height is needed to avoid zero-height element */
+      /* The min-height is same as form-control's default height */
         height: auto;
-        min-height: 26px;
-
+        min-height: em(26);
+        margin-bottom: 0;
         /* Helper class, should be moved out when almEditable is extended to support placeholders */
         &.placeholder {
-            color: $color-pf-black-400;
+            color: $color-pf-black-200;
```

Comment:
This color is very light and hardly visible with white background. Can we use the same placeholder color which we are using for description field?